widgets/src/google/CommonND.js

```diff
@@ -16,12 +16,28 @@
     CommonND.prototype.implements(INDChart.prototype);
 
     CommonND.prototype.publish("paletteID", "default", "set", "Palette ID", CommonND.prototype._palette.switch());
+    
+    CommonND.prototype.getChartOptions = function () {
+        var chartOptions = Common.prototype.getChartOptions.call(this);
+        chartOptions.series = initSeries(this.getNumSeries());
+        chartOptions.axes = {};
 
+        return chartOptions;
+    };
+    
     CommonND.prototype.update = function(domNode, element) {
-        this._palette = this._palette.switch(this._paletteID);
-
+  		this._palette = this._palette.switch(this._paletteID); 
```

Comment:
doesn't look like that on this end but will reformat that ares just to make sure